shared/rspec/support/capybara.rb

```diff
@@ -1,26 +1,46 @@
 require 'selenium-webdriver'
 require 'capybara'
+require 'capybara/rspec'
+
+# Give CI some extra time
+CAPYBARA_TIMEOUT = ENV['TEST_CI_SERVER'] ? 60 : 30
```

Comment:
While checking the CI services, I notice that most of it have `CI=true` variable set. Not sure, should we use that variable for checking or not?

Reply:
Great idea. Will change to this variable 👍